user/templates/user/data_transfer.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ super() }} &dash; Data Transfer{% endblock %}
+
+{% block scripts %}
+    <script src="{{ url_for('user_bp.static', filename='js/data_transfer.js') }}"></script>
+    <script src="{{ url_for('static', filename='lib/highlight-js-11.9.0/highlight.min.js') }}"></script>
+{% endblock %}
+
+{% block style %}
+    <link id='pagestyle' rel="stylesheet" {% if g.settings and g.settings.color_mode and g.settings.color_mode == 'DARK' %}href="{{url_for('static', filename='lib/highlight-js-11.9.0/styles/dark.min.css')}}"{% else %}href="{{url_for('static', filename='lib/highlight-js-11.9.0/styles/default.min.css')}}"{% endif %}>
```

Comment:
This does not work when the user has color mode set to auto and system theme to dark. Then the light css will be loaded.